packages/flutter_tools/lib/src/commands/build_fuchsia.dart

```diff
@@ -19,6 +19,14 @@ class BuildFuchsiaCommand extends BuildSubCommand {
   BuildFuchsiaCommand({bool verboseHelp = false}) {
     usesTargetOption();
     addBuildModeFlags(verboseHelp: verboseHelp);
+    argParser.addOption(
+      'runner-source',
+      help: 'The package source to use for the flutter_runner. '
+            '"fuchsia.com" implies using a runner in the installed Fuchsia image. '
+            '"flutter_tool" implies using a runner distributed with Flutter.',
+      allowed: <String>['fuchsia.com', 'flutter_tool'],
```

Comment:
nit: represent this as an enum/class or create const values to avoid stringly-typed issues.

Reply:
Done.